common/src/main/java/com/alibaba/nacos/common/http/client/NacosRestTemplate.java

```diff
@@ -298,7 +297,29 @@ public NacosRestTemplate(HttpClientRequest requestClient) {
                 Query.newInstance().initParams(paramValues), bodyValues);
         return execute(url, HttpMethod.POST, requestHttpEntity, responseType);
     }
-    
+
+    /**
+     * Execute the HTTP method to the given URI template, writing the given request entity to the request, and
+     * returns the response as {@link HttpRestResult}.
+     *
+     * @param url          url
+     * @param header       http header param
+     * @param paramValues  http query param
+     * @param bodyValues   http body param
+     * @param httpMethod   http method
+     * @param responseType return type
+     * @return {@link HttpRestResult}
+     * @throws Exception ex
+     */
+    public <T> HttpRestResult<T> exchangeFrom(String url, Header header,
```

Comment:
What's the mean of `From` I read the method think it should be `exchangeForm`, is it?